includes/Core/User/User.php

```diff
@@ -13,7 +13,7 @@
 use Google\Site_Kit\Core\Storage\User_Options;
 
 /**
- * Class for handling audience settings rest routes.
+ * Class for handling user settings rest routes.
```

Comment:
This class was originally added during Audience segmentation epic, but is actually intended to be used as a single entry point for all core/user settings that are added. See issue description on #8810